linera-service/src/cli/command.rs

```diff
@@ -750,6 +750,10 @@ pub enum ClientCommand {
         /// Path to the persistent storage file for faucet mappings.
         #[arg(long)]
         storage_path: Option<PathBuf>,
+
+        /// Maximum number of operations to include in a single block (default: 100).
+        #[arg(long, default_value = "100")]
+        max_batch_size: usize,
```

Comment:
Maybe it's safer to do it just for the faucet ?

Reply:
I don't understand: This argument _is_ for the client, isn't it?